### PR TITLE
fix: navigation, token icons and price

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/data/tokens.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
   "tokens": [
+    {
+      "chainId": 10,
+      "address": "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
+      "name": "USD Coin",
+      "symbol": "USDC",
+      "decimals": 6,
+      "logoURI": "https://assets.coingecko.com/coins/images/6319/standard/usdc.png?1696506694"
```
Comment (turtlemoji) [inline src/data/tokens.json:9]: `is ?1696506694 needed?` what happens if the url changes tho

File: src/providers/StateProvider.tsx [hunk 2]
```diff
@@ -170,11 +170,6 @@ export const StateProvider = ({ children }: StateProps) => {
     setVaults([]);
   }, []);
 
-  // Load vaults on load
-  useEffect(() => {
```
Comment (turtlemoji) [inline src/providers/StateProvider.tsx:174]: why removed?

File: src/data/tokens.json (hunk 1 above)
Comment (0xArdy) [reply src/data/tokens.json:9]: it isnt needed

Comment (0xArdy) [reply src/data/tokens.json]: removed https://github.com/defi-wonderland/xkeeper-app/pull/36/commits/6dd775b2aa330d67e39475db938640f380669db3

File: src/providers/StateProvider.tsx (hunk 2 above)
Comment (0xArdy) [reply src/providers/StateProvider.tsx:174]: now we check first the selected chain (in the search bar) before loading the vaults, so its moved to the page component

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**